test/modules/fundingManager/RebasingFundingManager.t.sol

```diff
@@ -104,7 +104,7 @@ contract RebasingFundingManagerTest is ModuleTest {
     // Tests: Initialization
 
     function testInit() public override(ModuleTest) {
-        assertEq(fundingManager.decimals(), DECIMALS);
+        //assertEq(fundingManager.decimals(), DECIMALS);
```

Comment:
also outcommented this code
probably in connection with the comment above

Reply:
Same as above, yeah.